.github/workflows/build-kernel-arcvm.yml

```diff
@@ -74,6 +74,17 @@ jobs:
           echo "VERSION: $VERSION"
           echo "kernelsu_version=$VERSION" >> $GITHUB_ENV
 
+      - name: Disable compiler warnings
+        working-directory: kernel
```

Comment:
I think this is a bad idea

Reply:
If adding KernelSU results in a compilation failure due to a warning, then we should fix the warning instead of disabling it.

Reply:
Removing the patch as fixes are available, hopefully the new changes can deal with it...